src/components/cart/cartProduct.js

```diff
@@ -0,0 +1,64 @@
+import './index.css'
+import {useState} from 'react'
+import {Button} from '../index'
+import {useCart} from '../../contexts/cartContext'
+import { GoX } from "react-icons/go";
+import axios  from "axios";
+
+const CartProduct=({product})=>{
+  const {dispatchCart,cartCount}=useCart()
+  const {title,imageUrl,price}=product
+  const[quantityPrice,setQuantityPrice]=useState(Number(price))
+  const [count,setCount]=useState(1)
+  const isdisable=count===1?true:false
+  const decreaseQunatity=(product)=>{  
+    setQuantityPrice((prevPrice)=>prevPrice-Number(product.price))
+    setCount((prevCount)=>prevCount-1)
```

Comment:
I liked the naming convention here while changing the state, you have used prevPrice , I used prevstate there and previous price is definitely a better name for that so that is good.